Library/Formula/curl.rb

```diff
@@ -47,7 +51,7 @@ def install
     if build.with?("libressl") && build.with?("openssl")
       ohai <<-EOS.undent
       --with-openssl and --with-libressl are both specified and
-      curl can only use one at a time; proceeding with openssl.
+      curl can only use one at a time; proceeding with libressl.
```

Comment:
Hmm, but this is *only* for the case where both `--with-openssl` and `--with-libressl` is defined. This makes the logic for deciding which lib to use much cleaner and also fixes a bug where openssl was used on pre Mountain Lion even if `--with-libressl`was specified. So I really think this is the much better choice.